src/commands/editor-quick-pick.ts

```diff
@@ -55,7 +57,10 @@ export default function createEditorQuickPickCommand(
             }
         });
 
-        quickPick.onDidHide(quickPick.dispose);
+        quickPick.onDidHide(() => { 
+            workspaceService.setQuickPickContext(false);
+            quickPick.dispose ;
```

Comment:
Looks like this is missing an invocation of the function

Reply:
Good pickup. I'll fix that right now.